pom.xml

```diff
@@ -683,11 +669,7 @@ It supports test-case and test-suite which are Selenium IDE's native format.</de
       <artifactId>jakarta.xml.bind-api</artifactId>
       <version>2.3.3</version>
     </dependency>
-    <dependency>
-      <groupId>org.glassfish.jaxb</groupId>
-      <artifactId>jaxb-runtime</artifactId>
-      <version>2.3.4</version>
-    </dependency>
```

Comment:
Please do not delete this part.
Removing this part will cause a runtime exception in Java 11 or later.